### PR TITLE
Add option 114 for RFC 8910 (auto show wizard screen on iOS 14/big sur)

### PR DESCRIPTION
**Pull request by petermm**
obviously building upon https://github.com/nerves-networking/vintage_net_wizard/pull/158

RFC 7710 was was superseded by 8910 and the option changed to 114. https://tools.ietf.org/html/rfc8910

best news is that iOS 14 and macOS big sur supports the option:
https://developer.apple.com/news/?id=q78sq5rv

NB: the ip address is used - since the client might have dns servers configured 1.1.1.1 google's or what not - disabling resolving of the hostname - ip works just fine and is not shown on the screens anyway.

basically it just works..

Screenshot 2021-02-11 at 18 51 39
IMG_5101

### CODE REVIEW

**Comment by fhunleth** [comment]
@petermm This is great news! But, I see that you closed the PR. Was that an accident?

**Comment by petermm** [comment]
unfortunately I spoke too soon, had messed up in my testing - so no accident - investigating further if this can be made to work...
